pycbc/inference/models/relbin.py

```diff
@@ -195,7 +209,13 @@ def __init__(
 
             # generate fiducial waveform
             f_lo = self.kmin[ifo] * self.df[ifo]
+
+            if is_hom:
```

Comment:
This seems a bit artificial here. Is there a reason this can't be set from the config file rather than hard coding the relation? There may be times we don't want to truncate at the same time for example.